agent/listener/static/help/index.html

```diff
@@ -64,17 +64,23 @@ <h3>NCPA Passive</h3>
                 <div class="section">
 
                     <h2>Restarting the NCPA Service</h2>
-                    <p>Because NCPA is made up of these two separate services, you will need to restart both of them if you are using both the active and passive capabilities. Below is the naming conventions and the command line way of restarting both services. In the configuration section you'll find that you will need to restart the services each time configuration changes are made.</p>
+                    <p>You will want to restart the NCPA service whenever a configuration change is made.</p>
```

Comment:
Maybe rephrase to indicate that they MUST restart the service for the changes to take effect.